app/src/main/java/com/duckduckgo/app/browser/BrowserTrackersAnimatorHelper.kt

```diff
@@ -201,6 +265,8 @@ class BrowserTrackersAnimatorHelper {
     companion object {
         private const val TRACKER_LOGOS_DELAY_ON_SCREEN = 2400L
         private const val DEFAULT_ANIMATION_DURATION = 150L
-        private const val MAX_LOGOS_SHOWN = 4
+        private const val MAX_LOGOS_SHOWN = 3
     }
-}
\ No newline at end of file
+}
+
+data class TrackerLogo(val resId: Int, val trackerLetter: String = "")
```

Comment:
I wonder if we should set empty as `trackerLetter` default value. Seems to me that we should enforce that value to the caller.

Reply:
As we talked in the call, his is used for both trackers with logos and without logos hence the use of a default value.